app/views/orders/_edit_date.html.haml

```diff
@@ -1,18 +1,27 @@
 - classes = ['collapsable']
 - classes << 'collapsed' if params['backdate_order'].blank?
-%fieldset#backdate-orders{ :class => classes }
+%fieldset#backdate-orders{ class: classes }
   %label.legend= t("orders.show.more_options")
   = hidden_field_tag :backdate_order, true
   - default_time = time_floor(Time.zone.now)
-  = simple_fields_for :backdate, :builder => ModelLessFormBuilder, :defaults => { :required => false } do |f|
-    = f.input :order_date, :input_html => { :value => params[:order_date] || format_usa_date(default_time), :class => 'datepicker' }
+  = simple_fields_for :backdate, builder: ModelLessFormBuilder, defaults: { required: false } do |f|
+    = f.input :order_date,
+      input_html: { value: params[:order_date] || format_usa_date(default_time),
+          class: "datepicker__data",
+          data: { max_date: Time.current.iso8601 } }
 
     = label_tag :order_time
     .time-select= time_select_tag :order_time, params[:order_datetime] || default_time
 
-    = f.input :order_status_id, :collection => @order_statuses, :label_method => :name_with_level, :selected => params[:order_status_id] || @order_statuses.first.try(:id), :label => OrderDetail.human_attribute_name(:order_status)
+    = f.input :order_status_id,
+      collection: @order_statuses,
+      label_method: :name_with_level,
+      selected:  params[:order_status_id] || @order_statuses.first.try(:id),
+      label: OrderDetail.human_attribute_name(:order_status)
 
-    = f.input :send_notification, :as => :boolean, :input_html => { :value => 1 }, :default => params[:send_notification], :label => false, :inline_label => true
-
-  :javascript
-    $(function() { $('.datepicker').datepicker({maxDate: '#{format_usa_date(Time.zone.now)}'}); });
```

Comment:
Our use of jquery-ui's datepicker is a little weird. We only call `$(".datepicker").datepicker()` on the views that need it. I believe we did this so we could control `maxDate` and `minDate` on a per-page level.

A while back, I decided that using data attributes would be a good solution so we could call `.datepicker()` from the main `application.js`. I went with `.datepicker__data` to avoid conflicts and to imply that it's meant to be used with data attributes. Not sure it's the perfect convention, but it works.

See `app/assets/javascripts/app/datepicker-data.coffee`. We load the whole `javascripts/app` tree into `application.js`.